test/int127_test.cpp

```diff
@@ -0,0 +1,70 @@
+#include <gtest/gtest.h>
```

Comment:
I think it's important to have comments explaining what each test does. Just a line or two should suffice.